config/bidderinfo.go

```diff
@@ -161,13 +161,13 @@ type SyncerEndpoint struct {
 	// startup:
 	//
 	//  {{.ExternalURL}} - This will be replaced with the host server's externally reachable http path.
-	//  {{.SyncerKey}}   - This will be replaced with the syncer key.
+	//  {{.BidderName}}   - This will be replaced with the bidder name.
```

Comment:
Nitpick: Please fix the white space so the `- This will be..` comment aligns with the rest.

Reply:
After the revert it went back and now aligns with other placeholders